fw/http_frame.c

```diff
@@ -2146,7 +2154,7 @@ tfw_h2_make_frames(struct sock *sk, struct sk_buff *skb, TfwH2Ctx *ctx,
 	limit -= FRAME_HEADER_SIZE;
 
 	frame_hdr.stream_id = stream->id;
-	frame_hdr.type = type;
+	frame_hdr.type = type == HTTP2_TRAILER_HEADERS ? otype : type;
```

Comment:
In case if you implement `tfw_h2_make_trailer_frames` toy don't need HTTP2_TRAILER_HEADERS and this part of code